lib/server/ServerRenderingWorkflow.js

```diff
@@ -131,6 +131,10 @@ function redirectOrLogAndRenderError(forRoute) {
 
         serverResponse.status(status);
 
+        if (!layout.hasBeenRendered) {
+            layout.render();
+        }
+
```

Comment:
before 0.47.0, the layout would always render BEFORE the view was chosen for the route. Now that the layout is rendering AFTER view is chosen, if there is an error while choosing the view, we can get a case where the layout hasn't been rendered. that's why we saw a blank page. these lines say, if we get here and haven't already rendered the layout, render it now